scripts/remove-low-rated-notes

```diff
@@ -23,16 +23,20 @@ $table = "Rating | Note\n"
        . "-------+---------------------------------------------------------\n";
 
 $count = 0;
+$noteIDs = [];
 foreach ($result as $row) {
     $table .= sprintf("%5d  | https://php.net/manual/en/%s.php#%s\n", $row['weight'], $row['sect'], $row['note_id']);
-	$count++;
+    $noteIDs[] = $row['note_id'];
+    $count++;
 }
 
 $body = "Following were the {$count} notes with a rating less than " . RATING_THRESHOLD . " and\nare older than " . AGE_THRESHOLD. ".\n\n"
       . sprintf("These notes represented %.1f%% of the %d total user notes,\nand have now been removed.\n\n", ($count / $total) * 100, $total)
       . $table;
 
-$query = "DELETE FROM note WHERE id IN (SELECT note.id FROM note, votes WHERE note.ts < ? AND note.id = votes.note_id GROUP BY note_id HAVING SUM(if (vote = 0, -1, 1)) < ?)";
-$result = $pdo->safeQuery($query, [ $date, RATING_THRESHOLD ]);
+foreach ($noteIDs as $noteID) {
+    $query = "DELETE FROM note WHERE id = ?";
+    $result = $pdo->safeQuery($query, [ $noteID ]);
```

Comment:
In this case, you should use `prepare` and `execute` instead of safeQuery. Prepare before the loop and execute within the loop.